integration-tests/utils/integration-utils.js

```diff
@@ -497,6 +522,46 @@ const verifyAccessibility = async (page, allowedViolations) => {
   }
 };
 
+/** Converts various time units to milliseconds. */
+const Millis = (() => {
+  const MS_IN_SEC = 1000;
+  const SEC_IN_MIN = 60;
+  const MIN_IN_HR = 60;
+  const NONE = 0;
+  const none = NONE;
+  const ofMillis = (millis) => +millis;
+  const ofMilli = ofMillis(1);
+  const ofSeconds = (seconds) => +seconds * ofMillis(MS_IN_SEC);
+  const ofSecond = ofSeconds(1);
+  const ofMinutes = (minutes) => +minutes * ofSeconds(SEC_IN_MIN);
+  const ofMinute = ofMinutes(1);
+  const ofHours = (hours) => +hours * ofMinutes(MIN_IN_HR);
+  const ofHour = ofHours(1);
+  const ofObject = ({ hours = 0, minutes = 0, seconds = 0, millis = 0 }) =>
+    ofHours(hours) + ofMinutes(minutes) + ofSeconds(seconds) + ofMillis(millis);
+  const of = (countOrHms) => {
+    if (typeof countOrHms === 'number') {
+      return ofMillis(countOrHms);
+    }
```

Comment:
Completeness (analogue to ofseconds/ofminutes) + comparability (if other timeouts are Millis.ofX, a Millis  value instead of a raw int self-documents the meaning of the value)